kerykeion/heliacal/heliacal_factory.py

```
@@ -198,32 +198,32 @@ def search_events(
+                        continue
+                    try:
+                        event = self._find_event(
+                            julian_day=julian_day,
+                            planet_name_or_star=planet,
+                            geopos=geopos,
+                            event_type=etype,
+                            atmo=atmo,
+                            observer=observer,
+                        )
+                        events.append(event)
+                    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)